src/main/resources/translator.properties

```diff
@@ -102,10 +126,10 @@ includeFailedTests = includeFailedTests
 includeFailedTests.type = com.adq.jenkins.xmljobtodsl.dsl.strategies.custom.DSLNonSupportedMethodDSL
 
 includeCustomMessage = includeCustomMessage
-includeCustomMessage.type = com.adq.jenkins.xmljobtodsl.dsl.strategies.custom.DSLNonSupportedMethodDSL
+#includeCustomMessage.type = com.adq.jenkins.xmljobtodsl.dsl.strategies.custom.DSLNonSupportedMethodDSL
 
 customMessage = customMessage
-customMessage.type = com.adq.jenkins.xmljobtodsl.dsl.strategies.custom.DSLNonSupportedMethodDSL
+#customMessage.type = com.adq.jenkins.xmljobtodsl.dsl.strategies.custom.DSLNonSupportedMethodDSL
```

Comment:
Let's remove instead of commenting out

Reply:
ok, they are on every attribute for slack so I will remove those